docs/references/express/overview.mdx

```diff
@@ -0,0 +1,165 @@
+const app = express()
+
+const port = 3000
+
+// Apply centralized middleware
+app.use(clerkMiddleware())
+
+// Define a protected route
+app.get('/protected', requireAuth, (req, res) => {
+  res.send('This is a protected route')
+})
+
+// Customize the error response through an error middleware
+app.use((err, req, res, next) => {
+  // Use the provided UnauthorizedError class to check if
+  // the error is an unauthorized error.
+  if (err instanceof UnauthorizedError) {
+    return res.status(401).send('Unauthorized')
+  }
+
+  return next(err)
+})
+
+// Start the server and listen on the specified port
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}`)
+})
+```
+
+### `getAuth()`
+
+The `getAuth()` helper retrieves authentication state from the request object. See the [Next.js reference documentation](https://clerk.com/docs/references/nextjs/get-auth) for more information on how to use it.
+
+The following example uses `requireAuth` to protect the route based on _authentication_ status, and then uses `getAuth()` to protect the route based on _authorization_ status.
+
+```js
+import { clerkMiddleware, getAuth, requireAuth } from '@clerk/express'
+import express from 'express'
+
+const app = express()
+const port = 3000
+
+// Apply centralized middleware
+app.use(clerkMiddleware())
+
+// Protect a route based on authorization status
+const hasPermission = (req, res, next) => {
+  const auth = getAuth(req)
+
+  // Handle if the user is not authorized
+  if (!auth.has({ permission: 'org:admin:testpermission' })) {
+    response.status(403).send('Forbidden')
+    return
+  }
+
+  return next()
+}
+
+app.get('/path', requireAuth, hasPermission, (req, res) => res.json(req.auth))
+
+// Customize the error response through an error middleware
+app.use((err, req, res, next) => {
+  if (err instanceof UnauthorizedError) {
+    return res.status(401).send('Unauthorized')
+  }
+
+  return next(err)
+})
+
+// Start the server and listen on the specified port
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}`)
+})
+```
+
+### `clerkClient`
+
+[Clerk's JavaScript Backend SDK](/docs/references/backend/overview) exposes Clerk's Backend API resources and low-level authentication utilities for JavaScript environments. For example, if you wanted to get a list of all users in your application, instead of creating a fetch to Clerk's `https://api.clerk.com/v1/users` endpoint, you can use the `users.getUserList()` method provided by the JavaScript Backend SDK.
+
+All resource operations are mounted as sub-APIs on the `clerkClient` object. See the [reference documentation](/docs/references/backend/overview#usage) for more information.
+
+```js
+import { clerkClient, requireAuth } from '@clerk/express'
+import express from 'express'
+
+const app = express()
+const port = 3000
+
+const users = await clerkClient.users.getUserList()
+
+app.get('/users', requireAuth, (req, res) => res.json(users))
```

Comment:
This example code would fetch the users list once when the server starts and return that same list every time someone hits this endpoint, even if users had been added or removed since. I think it would be better to recommend fetching every time, as this is the normal expected behavior of such a route.

```suggestion
app.get('/users', requireAuth, async (req, res) => {
  const users = await clerkClient.users.getUserList();
  return res.json(users)
})
```